src/charm.py

```diff
@@ -173,7 +209,10 @@ def _on_get_certificate_action(self, event: ActionEvent) -> None:
     def _on_certificates_relation_created(self, _: RelationCreatedEvent) -> None:
         """Handle the TLS Certificate relation created event."""
         tls_information = TLSInformation.from_charm(self)
-        config = CharmConfig.from_charm(self)
+
+        field_manager = self.app.name
+        namespace = self.model.name
+        config = CharmConfig.from_charm(self, get_client(field_manager, namespace))
```

Comment:
Minor: create a separate variable for `get_client` (also applies to line 228 and 337)

Reply:
Fixed!